recipes/tests.py

```diff
@@ -61,3 +62,188 @@ def test_same_name_ingredient(self):
     def test_list_ingredient(self):
         response = self.list_ingredient()
         self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+
+class ListCreateRecipeTestCase(TestCase):
+    url = reverse('list-create-recipe')
+
+    def create_user(self, username='testuser', email='testuser@mail.com'):
+        user = UserProfile.objects.create_user(username=username, email=email, password='testuser_1')
+        token, created = Token.objects.get_or_create(user=user)
+        client = APIClient()
+        client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+        return user, client
+
+    def create_ingredients(self, name='test_ingredient'):
+        ingredients = list()
+        ingredient = Ingredient.objects.create(name=name, image=base64image)
+        ingredient.save()
+        ingredients.append(ingredient.id)
+        ingredient = Ingredient.objects.create(name='test_ingredient_2', image=base64image)
+        ingredient.save()
+        ingredients.append(ingredient.id)
+        return ingredients
+
+    def create_recipe(self, client, title='test_title', description='test_desc', difficulty='E', image=base64image, any_ingredients=True):
```

Comment:
Why did you prefer to use `any_ingredients` flag instead of `ingredients` list ?